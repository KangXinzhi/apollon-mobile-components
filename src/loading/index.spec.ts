import { testComponent } from '../../test/utils'
import Loading from './index'

it('normal', () => {
  testComponent(Loading, wrapper => {
    expect(wrapper).toMatchSnapshot()
  }, { })
})
