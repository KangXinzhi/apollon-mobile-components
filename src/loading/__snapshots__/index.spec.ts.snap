// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`normal 1`] = `ShallowWrapper {}`;
